wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift

```diff
@@ -64,6 +65,7 @@ final class SelfProfileViewController: UIViewController {
     ) {
 
         self.userSession = userSession
+        accountSelector = SessionManager.shared
```

Comment:
This is in init right? Can the `accountSelector` be injected instead of using the singleton inside?

Reply:
Of course, thanks for the hint.